apps/ledger-live-desktop/src/renderer/families/celo/AccountBalanceSummaryFooter.js

```diff
@@ -64,19 +65,26 @@ const AccountBalanceSummaryFooter = ({ account, countervalue }: Props) => {
   const discreet = useDiscreetMode();
   const locale = useSelector(localeSelector);
 
-  const { spendableBalance: _spendableBalance } = account;
+  if (!account.celoResources) return null;
+
+  const { spendableBalance: _spendableBalance, celoResources } = account;
+  const { lockedBalance: _lockedBalance } = celoResources;
```

Comment:
Hey @pawelnguyen - why do we alias and prefix these vars with `_`?

Reply:
nice, good catch 👍 I started getting rid of it but didn't finish refactoring